tornjak-frontend/src/components/agent-list.tsx

```diff
@@ -71,7 +71,7 @@ class AgentList extends Component<AgentListProp, AgentListState> {
   TornjakApi: TornjakApi;
   constructor(props: AgentListProp) {
     super(props);
-    this.TornjakApi = new TornjakApi();
+    this.TornjakApi = new TornjakApi(props);
```

Comment:
why does tornjak api need props as args?

Reply:
Because the props we are utilizing for the API functions are "AgentListProp" types. Without it, it will throw an error.